dev-python/testpath/testpath-0.5.0.ebuild

```diff
@@ -16,13 +16,5 @@ LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sparc ~x86 ~x64-macos"
 
-BDEPEND="
-	>=dev-python/pyproject2setuppy-15
```

Comment:
That's not redundant. It doesn't work with older versions.

Reply:
Fixed.
Sorry for missing it.